src/spaced/spaced/scenes/game.cpp

```diff
@@ -48,16 +49,19 @@ void Game::on_loaded() {
 	m_player.foam_particles.set_texture(foam_texture);
 
 	auto const& rgbas = get_services().get<Styles>().rgbas;
-	auto spawn = [this] {
+	auto spawn = [this, rgbas] {
```

Comment:
Don't copy `rgbas` by value in the lambda, take it by reference instead; we know that the `Styles` reference will outlive `m_enemy_spawner` (because it's a service indirectly owned by `Spaced`) so it's safe. `[this, &rgbas]`